cypress/integration/app_layout.spec.js

```diff
@@ -0,0 +1,20 @@
+const { createArrayTypeNode } = require("typescript")
+
+describe('App layout', () =>{
+    it('Check footer presence', () =>{
```

Comment:
IMHO, this functionality (which pages show the footer) is too trivial to cover.
Tests run slowly, so we need to prefer more complex scenarios to fit in a reasonable time budget.